apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -249,6 +249,18 @@ defmodule OMG.ChildChain.BlockQueue.Core do
     :ok
   end
 
+  # https://github.com/ethereum/go-ethereum/commit/9938d954c8391682947682543cf9b52196507a88#diff-8fecce9bb4c486ebc22226cf681416e2
+  def process_submit_result(
+        submission,
+        {:error, %{"code" => -32_000, "message" => "already known"}},
+        _newest_mined_blknum
+      ) do
+    log_known_tx(submission)
+    :ok
+  end
+
+  # maybe this will get deprecated soon once the network migrated to 1.9.11. look at the previos function header
```

Comment:
```suggestion
  # maybe this will get deprecated soon once the network migrates to 1.9.11. Llook at the previos function header for commit reference.
```